homeassistant/components/shelly/utils.py

```diff
@@ -367,6 +367,11 @@ def is_rpc_channel_type_light(config: dict[str, Any], channel: int) -> bool:
     return cast(str, con_types[channel]).lower().startswith("light")
 
 
+def is_rpc_thermostat_internal_actuator(status: dict[str, Any]) -> bool:
+    """Return true if the thermostat uses an internal relay."""
+    return cast(bool, status.get("relay_in_thermostat", False))
```

Comment:
```suggestion
    return cast(bool, status["sys"].get("relay_in_thermostat", False))
```
Should be `status["sys"]`.

![obraz](https://github.com/home-assistant/core/assets/478555/656120d7-006f-4eb5-892e-10e564b1a70f)

Reply:
Overlooked, thx!